### PR TITLE
JENKINS-36641: e2e tests

### PR DESCRIPTION
**Pull request by imeredith**
# Description

See [JENKINS-36641JENKINS-36641](https://issues.jenkins-ci.org/browse/JENKINS-36641).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
@imeredith a lot to take in - what are the "cliff notes" of what is in this one?

File: acceptance-tests/src/test/java/io/blueocean/ath/offline/edgeCases/FolderTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                .pipeline(
+                    Folder.folders("aFold er", URLEncoder.encode("bF older"), "cFolder"),
+                    pipelineName)
+                .createPipeline(git);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw e;
+        }
+        ActivityPage activityPage = p.getActivityPage();
+        activityPage.open();
```
Comment (michaelneale) [inline acceptance-tests/src/test/java/io/blueocean/ath/offline/edgeCases/FolderTest.java]: isn't this meant to do some assertions at this point?

**Comment by cliffmeyers** [comment]
Is it expected that the tests should be passing? I ran with `--dev` flag and then `mvn test` and got two failures:

```
INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   CommitMessagesTest.commitMessagesTest:72 » Timeout Timed out after 60 seconds ...
[ERROR]   ParallelNavigationTest.parallelNavigationTest:71 » Timeout Timed out after 10 ...
[INFO] 
[ERROR] Tests run: 5, Failures: 0, Errors: 2, Skipped: 0
```

**Comment by cliffmeyers** [comment]
These seem pretty flaky (see below)? Also seems like it's opening a new browser tab / window for every test in the Docker container... kind of impossible to see what's going on.

```
ERROR] Errors: 
[ERROR]   ParallelNavigationTest.parallelNavigationTest:71 » Timeout Timed out after 10 ...
[ERROR]   ParallelNavigationTest.parallelNavigationTestInput:93 » Timeout Timed out afte...
```

**Comment by imeredith** [comment]
@cliffmeyers Easiest way to keep browser open is to set a break point.

parallelNavigationTest failing makes me think you might not have rebuilt the hpi's because that test is for something micheal fixed yetsterday

The main worrying thing for me is the commit messages test

**Comment by michaelneale** [comment]
Ivan why is there also a branch with the same name on the origin repo as well as your fork?

**Comment by imeredith** [comment]
@cliffmeyers sorry should have left a note. This branch is very much a WIP. Somehow since i merged micheals tests, everything is flaky.  But only server side for me. Everything works locally. Trying to get to the root of the issue

**Pull request merged**